internal/auth0/anomaly.go

```diff
@@ -1,3 +1,5 @@
+//go:generate mockgen -source=anomaly.go -destination=anomaly_mock.go -package=auth0
```

Comment:
are we using this new mock already? If not, I prefer to avoid its generation

Reply:
Ok, I'll remove the mocks.

Reply:
Done. The new ones were removed in https://github.com/auth0/auth0-cli/pull/316/commits/46cb25c7903ba70db43d8a58f7fb24bdebb45a89, and the modified ones (because of different version of mockgen) were reverted in https://github.com/auth0/auth0-cli/pull/316/commits/e1cf52dfea62b9fc7a456293b03b466f2842d95e and https://github.com/auth0/auth0-cli/pull/316/commits/675930fad27b53ea634b529b4aed0252ee929817.

Reply:
great work, thanks